src/features/documentation.ts

```diff
@@ -29,3 +30,53 @@ export function extractSummaryText(xmlDocComment: string): string {
 
     return summary.slice(0, endIndex);
 }
+
+export function GetDocumentationString(structDoc: protocol.DocumentationComment) {
+    let newLine = "\n\n";
+    let indentSpaces = "\t\t";
+    let documentation = "";
+    if (structDoc == null) {
+        return documentation;
+    }
+    if (structDoc.SummaryText) {
+        documentation += structDoc.SummaryText + newLine;
+    }
+
+    if (structDoc.TypeParamElements && structDoc.TypeParamElements.length > 0) {
+        documentation += "Type Parameters:" + newLine;
+        documentation += indentSpaces + structDoc.TypeParamElements.map(displayDocumentationObject).join("\n" + indentSpaces) + newLine;
+    }
+
+    if (structDoc.ParamElements && structDoc.ParamElements.length > 0) {
+        documentation += "Parameters:" + newLine;
+        documentation += indentSpaces + structDoc.ParamElements.map(displayDocumentationObject).join("\n" + indentSpaces) + newLine;
+    }
+
+    if (structDoc.ReturnsText) {
+        documentation += structDoc.ReturnsText + newLine;
+    }
+
+    if (structDoc.RemarksText) {
+        documentation += structDoc.RemarksText + newLine;
```

Comment:
Might be worth extracting out a little helper method that adds the newline for you.